regulations/static/regulations/css/less/module/footer.less

```diff
@@ -138,7 +138,7 @@ Link Styles
 
 .next-prev-link:link,
 .next-prev-link:visited {
-    color: @pacific;
+    color: @link_blue;
```

Comment:
Something's fishy here -- the external links don't match:

<img width="288" alt="screen shot 2016-06-22 at 7 18 59 pm" src="https://cloud.githubusercontent.com/assets/326918/16287019/3bb645b8-38ae-11e6-8eef-0a879d6f94fc.png">